### PR TITLE
TimelineLog

### PR DESCRIPTION
**Pull request by tomchop**
This PR introduces tracking of changes in objects across time.

* A new schema containing timestamps, actors, and details of the changes is created in a new collection
* A new API endpoint is to retrieve this information, based on an object ID

### CODE REVIEW

File: core/web/apiv2/audit.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+from fastapi import APIRouter, HTTPException
+from pydantic import BaseModel, ConfigDict
+
+from core.schemas.audit import TimelineLog
+
+router = APIRouter()
+
+
+@router.get("/timeline/{id:path}")
```
Comment (udgover) [inline core/web/apiv2/audit.py:11]: In order to not confuse users, we need to detail in the documentation that the whole path will also include the collection name like `entities/{id}`

Comment (udgover) [reply core/web/apiv2/audit.py:11]: I also wonder how the OpenAPI spec will behave when building language specific client.

**Review by udgover — APPROVED** [review]
Really nice PR! Added one comment regarding timeline endpoint.

I also wonder if we should not log deletion as well and have a dedicated view for all deleted objects since we can no longer bind it to the object itself.

**Review by udgover — APPROVED** [review]
LGTM!

**Comment by tomchop** [comment]
This PR fixes #1195

**Pull request merged**